src/trace-plugin-interface.js

```diff
@@ -184,15 +157,14 @@ PluginAPI.prototype.runInRootSpan = function(options, fn) {
 };
 
 /**
- * Creates and returns a new ChildSpan object nested within the root span object
- * returned by getRootSpan. If there is no current RootSpan object, this
- * function returns null.
+ * Creates and returns a new ChildSpan object nested within the root span.If
+ * there is no current RootSpan object, this function returns null.
  * @param {object} options An object that specifies options for how the child
  * span is created and propogated.
  * @returns A new ChildSpan object, or null if there is no active root span.
  */
 PluginAPI.prototype.createChildSpan = function(options) {
-  var rootSpan = this.getRootSpan();
+  var rootSpan = this.getRootSpan_();
```

Comment:
Do you think it would be better to make this module-private (like `createRootSpan_`)?